internal/distro/rhel8/distro.go

```diff
@@ -691,12 +691,12 @@ func (t *imageType) checkOptions(customizations *blueprint.Customizations, optio
 	return nil
 }
 
-// New creates a new distro object, defining the supported architectures and image types
-func New() distro.Distro {
+// NewRHEL83 creates a new distro object, defining the supported architectures and image types
+func NewRHEL83() distro.Distro {
```

Comment:
I don't understand this. I though we will get rid of minor version, except for EUS. This does the opposite. I am very confused by this.

Reply:
There is also considerable overlap with #2830

Reply:
my mistake